node_modules/oae-principals/tests/data/users-with-password-alias.csv

```diff
@@ -0,0 +1,25 @@
+user-zfdHliPLa,password1,Matthijs,Nicolaas,nicolaas@alias.test.com
+user-plOPperDe,password2,Pareyn,Bert,bert@alias.test.com
+user-mIAUwkeNS,password3,Visser,Branden,branden@alias.test.com
+user-HlKKreaDP,password4,Gaeremynck,,simon@alias.test.com
+user-IrewPDSAw,password5,Freeman,Stuart D.,stuart@alias.test.com
+user-JjuyZssJQ,password6,Reynolds,Malcom,mal@alias.test.com
+user-HgRNdDzQd,password7,Washburn,Zoe,zoe@alias.test.com
+user-DGdlFUNzv,password8,Washburn,Hoban,wash@alias.test.com
+user-gPvjYjVHh,password9,Serra,Inara,inara@alias.test.com
+user-TUolZGzsm,password10,Cobb,Jayne,jayne@alias.test.com
+user-CIwpSRwXk,password11,Frye,Kaylee,kaylee@alias.test.com
+user-SUMbWTWKD,password13,Tam,River,river@alias.test.com
```

Comment:
Most apartment buildings skip the 13th floor, this is the first superstition I've seen for the number 12, though :)